src/database/schema/ChannelListSchema.ts

```diff
@@ -426,7 +426,7 @@ class ChannelList implements BaseDbSchema {
       description: descriptionSystemMessage || firstMessage?.text || firstMessage?.message || '',
       unreadCount: data?.unreadCount ?? 0,
       channelType,
-      lastUpdatedAt: data?.last_message_at,
+      lastUpdatedAt: data?.last_message_at ?? data?.created_at,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It ensures that if `last_message_at` is not available, the `created_at` timestamp will be used as a fallback for `lastUpdatedAt`. This can prevent potential issues where `lastUpdatedAt` might be undefined.
```diff
- lastUpdatedAt: data?.last_message_at,
+ lastUpdatedAt: data?.last_message_at ?? data?.created_at,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->